core/unit_test/incremental/Test12a_ThreadScratch.hpp

```diff
@@ -74,9 +74,15 @@ struct ThreadScratch {
     for (int i = 0; i < sY; ++i) v_S(i) = 0;
 
     Kokkos::parallel_for(Kokkos::TeamThreadRange(team, sX), [&](const int m) {
+    // FIXME_SYCL This deadlocks in the subgroup_barrier when running on CUDA
+    // devices.
```

Comment:
I would expect it to be an issue when compiling for the `CUDA` backend in `SYCL` (and thus the generated code) rather than that it's related to `NVIDIA` GPU's directly.
Either way, I am happy to change the wording if that is preferred.

Reply:
So that is SYCL terminology and not Kokkos, is that right?

Reply:
I see more `CUDA` than `NVIDIA` in git@github.com/intel/llvm if that is the question. The specifications don't mention any of that, of course.
In this context, I use `CUDA` and `NVIDIA` GPUs as synonyms. Again, I'm happy to change this WIP comment.

Reply:
You should open an issue on this since a deadlock indicated a bug in the code or in the instructions generated.

Reply:
See https://github.com/kokkos/kokkos/issues/4204.